### DISCUSSION TITLE
Retirement of Lane Direction Tool

### DISCUSSION THREAD

**Discussion by slyh** [slyh/Cities2-TrafficLightsEnhancement#96] [Announcements]
> [!NOTE]
> This is not a support thread. If you encounter any issues, whether related to the Traffic mod or not, please [open a new issue](https://github.com/slyh/Cities2-TrafficLightsEnhancement/issues/new/choose).

The Lane Direction Tool was developed as a stop-gap solution to provide players some control over junctions. With the release of the Traffic Mod from krzychu124, it has fulfilled its duty and is time for retirement. Traffic Lights Enhancement will continue to be developed with a focus on aspects of traffic lights.

Don't worry, all the lane settings set in LDT can be automatically converted to the Traffic Mod. Please follow this guide to ensure that you will not lose any settings.

1. Do not save any cities before installing the Traffic Mod by krzychu124.

2. Keep Traffic Lights Enhancement installed and install [the Traffic Mod from krzychu124](https://mods.paradoxplaza.com/mods/80095/Windows).

3. Load any saves with custom lane settings.

4. The Traffic Mod will inform you how many junctions have been converted.

5. Save your city, preferably in a new save file so that you have a backup if anything goes wrong.

6. Repeat the above steps for all your saves.

Traffic Lights Enhancement is not required for the Traffic Mod to function. We strive to ensure compatibility between both mods. You are free to keep or remove the Traffic Lights Enhancement after conversion.

Thanks to krzychu124 for developing the conversion feature.

## FAQ

**How long will the migration window be open?**

At least until the arrival of the next patch. I would recommend users complete the migration as soon as possible.

**What if I don't complete the migration on time?**

Your custom lane settings in Traffic Lights Enhancement might be lost, requiring you to set up the lane connections again.

**Comment by fowntain** [comment]
womp womp

**Reply by ssv2** [reply]
im confused, this is how open source works? A bunch of part timers that have day jobs and things? Do just like being first in line with fake internet points?

**Reply by bluefield-creator** [reply]
@ssv2 No, they just find something they wanna fix and share it with other folks. Who may or may not wanna help

**Comment by sealion200** [comment]
thank you for made tuis mod it vary useful !!

**Comment by JasonStephen** [comment]
Your mod is very useful, especially for our developed MIO Road Asset Pack. It greatly assists in creating primary and secondary road segregated lanes and shifting left turn lanes, and setting up directional signage. I am the author of the BRT road section of the MIO Road Asset Pack. Thank you for your contribution to Cities Skyline 2 and the Paradox Mods community. I wish you all the best in creating more excellent modules. ❤

**Comment by Twotoolus-FLY-LShst** [comment]
thank you keep support mod =D

**Comment by chameleon-tbn** [comment]
Thanks for working together on these things! Can you maybe contact me via discord to check if anything needed to be changed in the Compatibility Information of Skyve?

**Reply by slyh** [reply]
Hi Chamëleon,

I think the information on Skyve is accurate. The mod has not reached feature freeze and is still in the experimental phase. Once the next patch arrives, I'll remove my LaneSystem. This should allow the warning to be changed from "highly experimental" to "experimental".

**Comment by Me1onSeed** [comment]
恭喜毕业，呜呜呜TAT，TLE完成了它的历史使命！

**Reply by slyh** [reply]
While LDT has graduated, TLE still has a long way to go.

**Comment by Quazar000** [comment]
Hi!
I've installed TLE few days ago. And when I try to deactivate TLE I get a critical error. Can you help me?
It started today when I installed Traffic Mod

[SceneFlow] [CRITICAL]  System update error during Modification4->TrafficLaneSystem: System.MissingMethodException: Method not found: Unity.Collections.NativeQueue`1<Game.Net.Lane> Game.Net.LaneReferencesSystem.GetSkipLaneQueue()
  at Unity.Entities.SystemBase.Update () [0x0004e] in <56fb3e45f1bf4afcb7a333c1fffaae1f>:0 
  at Game.UpdateSystem.Update (Game.SystemUpdatePhase phase) [0x0004e] in <407a0a0b0b484732bd578acd1adf3d7c>:0 
0x00007ffa7cddedbd (UnityPlayer) UnityMain
0x00007ffa7cddeb38 (UnityPlayer) UnityMain
0x00007ffa7cdc5539 (UnityPlayer) UnityMain
0x00007ffa7d463cb1 (UnityPlayer) UnityMain
0x00007ffa7c44aca5 (UnityPlayer) 
0x000001e21252ee43 (Mono JIT Code) (wrapper managed-to-native) UnityEngine.DebugLogHandler:Internal_Log (UnityEngine.LogType,UnityEngine.LogOption,string,UnityEngine.Object)
0x000001e21252ed5b (Mono JIT Code) UnityEngine.DebugLogHandler:LogFormat (UnityEngine.LogType,UnityEngine.Object,string,object[])
0x000001e210befaa2 (Mono JIT Code) Colossal.Logging.CustomLogHandler:PostProcessFormat (UnityEngine.LogType,Colossal.Logging.Level,UnityEngine.Object,string,Colossal.Logging.ILog,System.Exception)
0x000001e210bef29b (Mono JIT Code) Colossal.Logging.CustomLogHandler:LogFormat (UnityEngine.LogType,UnityEngine.Object,string,object[])
0x000001e210beef61 (Mono JIT Code) UnityEngine.Logger:LogFormat (UnityEngine.LogType,string,object[])
0x000001e210bee91d (Mono JIT Code) Colossal.Logging.UnityLogger:Log (Colossal.Logging.Level,string,System.Exception)
0x000001e33ccf164b (Mono JIT Code) Colossal.Logging.UnityLogger:CriticalFormat (System.Exception,string,object,object)
0x000001e27df3527d (Mono JIT Code) Game.UpdateSystem:Update (Game.SystemUpdatePhase)
0x000001e27e234f6b (Mono JIT Code) Game.Common.ModificationSystem:OnUpdate ()
0x000001e27df3a6db (Mono JIT Code) Unity.Entities.SystemBase:Update ()
0x000001e27df351af (Mono JIT Code) Game.UpdateSystem:Update (Game.SystemUpdatePhase)
0x000001e27420c4c3 (Mono JIT Code) Game.SceneFlow.GameManager:UpdateWorld ()
0x000001e215d75cd3 (Mono JIT Code) Game.SceneFlow.GameManager:Update ()
0x000001e1b66f2bb8 (Mono JIT Code) (wrapper runtime-invoke) object:runtime_invoke_void__this__ (object,intptr,intptr,intptr)
0x00007ffa96134c2e (mono-2.0-bdwgc) mono_jit_set_domain
0x00007ffa9606d254 (mono-2.0-bdwgc) mono_object_get_virtual_method
0x00007ffa9606d3cc (mono-2.0-bdwgc) mono_runtime_invoke
0x00007ffa7cb4d8e6 (UnityPlayer) 
0x00007ffa7cb4da11 (UnityPlayer) 
0x00007ffa7c81f930 (UnityPlayer) 
0x00007ffa7c98c4d8 (UnityPlayer) 
0x00007ffa7c98c6c4 (UnityPlayer) 
0x00007ffa7c98cbe2 (UnityPlayer) 
0x00007ffa7cbde9db (UnityPlayer) 
0x00007ffa7cbe13ab (UnityPlayer) 
0x00007ffa7cbe470b (UnityPlayer) 
0x00007ffa7cbe49ab (UnityPlayer) UnityMain
0x00007ff671a911f2 (Cities2) 
0x00007ffae0c47374 (KERNEL32) BaseThreadInitThunk
0x00007ffae0d9cc91 (ntdll) RtlUserThreadStart

**Reply by krzychu124** [reply]
Simply update the game to the latest version. The latest version of _Traffic_ mod (error comes from that mod) is compatible only with the CSII 1.2.5+ and you are still on something much older (_1.2.5 was released 18th March 2025_).

**Reply by Quazar000** [reply]
Thank you so much!

**Comment by jay2jay99** [comment]
I just installed the Traffic Light enhancement tool but it keeps crashing my game, I had the traffic mod installed first and have a save just with that, I installed the traffic light enhancement tool and start configuring junctions then within 5min the game crashes back to my desktop, is there a certain order it has to be installed in? I'm on the latest version.

**Comment by slyh** [comment]
Could you share your Player.log?

One common cause of persistent crashes is having the BepInEx version of TLE
installed.

**Comment by jay2jay99** [comment]
[Player.log](https://github.com/user-attachments/files/20589142/Player.log)

I think I may have fixed it now, not sure why but will share the player log and a history of what I did in case it helps. All installs of base software and mods were recent.

I recently installed Cities Skylines 2 after a new install of windows (a hadn't played it for a while before that), I played the game initially with a few mods that I saw I had used previously and then installed the traffic mod after playing and saving a few times, all worked fine, A few days later I installed the Traffic Light Enhancement tool and after about 5-6min the game would just crash to the desktop every time with no error message. I disabled TLE but it still crashed. I deleted the contents of the cache folder and let everything reinstall from scratch, played with Traffic mod and TLE enabled, the game crashed again. I disabled both mods, played the game and it didn't crash..... I saved the game after a bit of progress. When you message I thought I'd re-enable the mods to trigger the crash again but now it's not crashing....odd. Whether the save with neither mod helped, I don't know.

**Reply by slyh** [reply]
I see you don't have BepInEx installed, so the problem isn't caused by an outdated version of TLE.

Your game hit a strange error, but it seems to be able to recover?

```
=================================================================
Native Crash Reporting
=================================================================
Got a UNKNOWN while executing native code. This usually indicates
a fatal error in the mono runtime or one of the native libraries
used by your application.
=================================================================

=================================================================
Managed Stacktrace:
=================================================================
 at <unknown> <0xffffffff>
 at UnityEngine.Object:GetName <0x00089>
 at UnityEngine.Object:get_name <0x0001a>
 at Game.Prefabs.PrefabSystem:AddPrefab <0x004a2>
 at Game.Prefabs.AssetCollection:AddPrefabsTo <0x000aa>
 at Game.SceneFlow.AssetLibrary:Load <0x00252>
 at <LoadAssetLibraryAsync>d__150:MoveNext <0x0050a>
 at MoveNextRunner:InvokeMoveNext <0x00091>
 at System.Threading.ExecutionContext:RunInternal <0x001dd>
 at System.Threading.ExecutionContext:Run <0x0002a>
 at MoveNextRunner:Run <0x000ea>
 at YieldAwaiter:RunAction <0x00039>
 at WorkRequest:Invoke <0x00023>
 at UnityEngine.UnitySynchronizationContext:Exec <0x001b2>
 at UnityEngine.UnitySynchronizationContext:ExecuteTasks <0x0007a>
 at System.Object:runtime_invoke_void <0x00084>
=================================================================
```

If you encounter crashes again, feel free to [open an issue](https://github.com/slyh/Cities2-TrafficLightsEnhancement/issues/new/choose) and share your log and save files so I can try to reproduce it.

**Comment by BrotherWill** [comment]
Thank you completely fucking my save game. I can't get it to open at all now. Just toss the last week of work out the window.

**Reply by krzychu124** [reply]
What do you mean? Mentioned _Retirement of Lane Direction Tool_ took place **over a year ago**!

**Reply by seoidau** [reply]
Actually I'm glad I have read the comments and now seen the date of this, because...

My city now crashes CSII to desktop three seconds after opening it, some 40 hours after I last saved it and everything was fine. All 10 backups crash the same way. I have filed a [report about this](https://forum.paradoxplaza.com/forum/threads/saved-city-will-not-open-with-crash-to-desktop-back-to-steam.1862975/) in the Paradox Forum.

The only change or update I am aware of between closing and attempting to resume editing my city is that a notification popped up about this feature retirement when I first opened CSII.

...But now when I look at the date, why was I notified about this a few days ago when it is over a year after it happened?!

Like @BrotherWill my previously-fine city now crashes CSII to desktop approx. three seconds after I attempt to open it. Not blaming you for this, but as I said it is the only thing that I noticed had changed.

**Reply by krzychu124** [reply]
> Actually I'm glad I have read the comments and now seen the date of this, because...
> 
> My city now crashes CSII to desktop three seconds after opening it, some 40 hours after I last saved it and everything was fine. All 10 backups crash the same way. I have filed a [report about this](https://forum.paradoxplaza.com/forum/threads/saved-city-will-not-open-with-crash-to-desktop-back-to-steam.1862975/) in the Paradox Forum.
> 
> The only change or update I am aware of between closing and attempting to resume editing my city is that a notification popped up about this feature retirement when I first opened CSII.
> 
> ...But now when I look at the date, why was I notified about this a few days ago when it is over a year after it happened?!
> 
> Like @BrotherWill my previously-fine city now crashes CSII to desktop approx. three seconds after I attempt to open it. Not blaming you for this, but as I said it is the only thing that I noticed had changed.

I tested the savegame you attached to bug report and it looks like some mod corrupted the data (most likely one of roads) and then it got saved so now the game is not able to read the data.
More info: 
```cs
=================================================================
	Managed Stacktrace:
=================================================================
	  at <unknown> <0xffffffff>
	  at Unity.Collections.LowLevel.Unsafe.UnsafeUtility:MemCpy <0x000f6>
	  at Unity.Entities.BufferHeader:SetCapacity <0x00302>
	  at Unity.Entities.BufferHeader:EnsureCapacity <0x00182>
	  at Unity.Entities.DynamicBuffer`1:EnsureCapacity <0x00112>
	  at Unity.Entities.DynamicBuffer`1:ResizeUninitialized <0x0007a>
	  at Unity.Entities.DynamicBuffer`1:Add <0x000da>
	  at SubBlockJob:Execute <0x0047a>
	  at SubBlockJob:Unity.Entities.IJobChunk.Execute <0x00092>
	  at JobChunkProducer`1:ExecuteInternal <0x00852>
	  at JobChunkProducer`1:Execute <0x0008a>
	  at ExecuteJobFunction:invoke_void_JobChunkExtensions/JobChunkWrapper`1<T>&_intptr_intptr_JobRanges&_int <0x00256>
	  at <Module>:runtime_invoke_void__this___intptr&_intptr_intptr_intptr&_int <0x00208>
=================================================================
```
`SubBlockJob` is a task running from `SubBlockSystem`, it is reading savegame data and cannot read the `Owner` of one of `SubBlock` entities. First broken entity is in chunk 37, 12. entity, the `Owner` with `Entity.Null` so refence to the road is completely missing, hence the game is crashing trying to store zone block reference using that invalid owner reference - **it should never happen** because every zoning block entity always has the `Owner` as zoning blocks are attached to road instance.

I see 6 broken zoning blocks located there (more or less): 
<img width="1664" height="868" alt="image" src="https://github.com/user-attachments/assets/d1d77f92-a683-45be-a1da-610b403586dd" />

<img width="1342" height="872" alt="image" src="https://github.com/user-attachments/assets/af411349-23e9-4555-8230-ab3732a70460" />

```cs
|| Block Entity  |                      (x,y,z)                      |        size       ||
Entity(1160807:1), position: float3(4308,738f, 523,6383f, -3608,158f) blockSize: int2(8, 6)
Entity(1164830:1), position: float3(4265,403f, 523,6383f, -3534,985f) blockSize: int2(8, 6)
Entity(1165020:1), position: float3(4387,307f, 523,6383f, -3557,691f) blockSize: int2(7, 6)
Entity(1166415:1), position: float3(4328,321f, 523,6383f, -3546,704f) blockSize: int2(8, 6)
Entity(1166452:1), position: float3(4249,752f, 523,6383f, -3597,171f) blockSize: int2(7, 6)
Entity(1167257:1), position: float3(4371,655f, 523,6383f, -3619,878f) blockSize: int2(8, 6)
```
Do you remember if you used any specific mod in this area? Any mod that is changing how the game zoning works? Maybe something touching that road network in any way?
I'm pretty sure it's not a vanilla game bug, but some mod silently corrupted the data before you saved.

**Reply by seoidau** [reply]
OH WOW! You were able to open it?! The buildings there are just initial development of mostly mixed residential zoning on the main road to get some life happening, and the only mod I would have used would have been the lane connector to turn off U-turns and unsafe connections which I always do at tram stops and intersections. I doubt I would have done much else, there is nothing special going on there that required special zoning. There didn't appear to be anything going wrong or not right when the game was playing. I did use the 'More Tram Stops' mod if there is a tram stop at that intersection, but that was a few good saves ago without issue.

If that's the case, ironic that this modding and this Retirement of Lane Direction Tool feature retirement notification has occurred, no? Again, how has this happened over a year after the feature has been retired (and I would have done that lane editing with Traffic Mod).

Are you able to blow away whatever in the game is causing my grief so I can continue to build the city? It's early, ugly work in that part of town and I would not miss it. This is the most help I have received for this -- thankyou and how can I buy you a few coffees or something? :)

**Reply by krzychu124** [reply]
I don't remember what conditions are set for showing notification but I've seen it many times when I was restarting the game while testing your savegame. 
The main question is: **can you break the savegame with your current set of mods?** Like, start new game or load one that works, play for a while and after saving under a new file the game won't be able to load it?

I'd like to know if it's ongoing problem or now everything seems to be working correctly (in case of loading older working savegames or starting new game). If you can still "generate" corrupted savegames, then it's definitely worth searching which mod is it, because more people can be affected and it's really hard to tell if two cases of crashing on load have the same source or not.

**Reply by seoidau** [reply]
I can't open the game – at all, ever. It CS2 crashes to desktop approx. three seconds after it starts to load it. No errors or messages appear, nothing.

I have reinstalled the game twice. Right now I have no mods installed at all, it is just the base game. I have a dummy game that I have opened, edited, saved, closed, reopened, edited, saved, closed, etc. a number of times over a few days and the game works fine. There is even an older huge city I had been working on (315MB) that I reopened and it was fine (gave up on it as I lost all residential demand and couldn't get anything back, plus a large homeless issue whilst the game said I had hardly any homeless, and I was over it). But, just like right now, when I attempt to open my corrupted city it crashes to desktop after approx. three seconds. To see you have been able to open it really surprised me!

According to Steam I have played CS2 1120 hours and I haven't had an issue like this before. I've even been rather lucky with few problems generally. Did get the "murky ocean" issue once when this corrupted city first began earlier this week.

Feel free to use it if it helps to find out the ultimate cause and help everyone! :)

**Reply by krzychu124** [reply]
You won't be able to open broken saved game(s), because vanilla data is corrupted (doesn't require mod to be loaded) and I had to create a mini mod to be able to load it but it's still broken and throws errors after loading. 
The question was if you can create similar broken saved game file just by playing the game normally, with the same or similar set of mods? 
That save file was almost definitely broken by a mod, the question is: _which one?_ 
If you now use less mods and things work correctly (when starting from scratch/new game), then it's most likely a mod you had before but you no longer have it subscribed. 
It's more about what mods you had when the save file was created than what mods you have now trying to load that saved file again (there's only one or two mods known that must be enabled or you won't be able to load the savegame).

I was suspecting _Station Naming_ mod since after looking into the source code it doesn't look great, there are comments about issues, but I couldn't find anything obvious which could corrupt the savegame data. 

You really didn't use any experimental mod, maybe a mod that changes how game zoning works (zoning tiles, maybe hiding) or maybe overusing _Move It_ mod by moving buildings or roads around too much?

**Reply by seoidau** [reply]
_Station Naming_ I did use and I believe I deactivated it shortly before all this because  annoyingly it renamed everything and I did read that it was problematic.

_Move It_ I have used sparingly and am aware it has caused problems with others. I don't recall using it at that area you circled. I did use it recently a little 'south' of that intersection – down in the corner of the area where you circled you'll see a very tight single-lane highway that takes traffic from a ring road highway to off the map, I used Move It a little to soften the tightness of the curve right before the edge of the map.

Other mods I was using:

- All Aboard! (Faster Boarding Mod)*
- Vibrant Foliage Pack*
- 529 Tiles
- Unlimited Unique Buildings
- More Tram Stops and Bus
- No Dead Trees
- Tram Variation Asset Pack
- Better Bulldozer
- Asset Icon Library
- Asset Packs Manager
- (Depreciated) Extended Cargo Harbour for Left-Hand Traffic [but I don't think I actually used it in the game]
- Traffic Lights Enhancement Alpha
- Traffic
- i18n EveryWhere
- Advanced Line Tool
- Unified Icon Library
- Find It

If I am to create a new map with the above mods then it would be a lot of editing to try to recreate the corruption, especially if there is nothing I have used in that circled area except for Lane Direction Tool, possibly a tram stop.

Only those above with an asterisk* are mods I have just started using with this city, plus Station Naming. The others I have used for some time, like with that 315MB city I was building previously, and not had any real significant issues like I have with this. To build another city hoping to recreate a corruption crash could be quite a lot of building hoping it will happen again..?

**Reply by krzychu124** [reply]
No need to build a new city just to try recreating corruption crash. I don't want to waste your time. 
I was curious if it's still happening, fairly easily. If it was somehow related with use of _Move It_ mod then it will definitely be very difficult to recreate. 

Since we are talking only about 3 mods, I'll take a look in their source code (assuming it's public) and see if there's anything suspicious that could cause the corruption. 

The main problem here is that something corrupted vanilla data and the game successfully saved that corrupted state. Now there's no way to point out what it was. If this could be replicated relatively easily (with a known set of steps to follow), it would make it much easier to narrow down where the source is.

Btw, I'm assuming that you don't have any a bit older saved game that can be loaded, with that circled area more or less the same as it was in corrupted one?

**Reply by seoidau** [reply]
I only have one version of this city with 10 backups and they all crash. Hadn't worked on it too long, but long enough that I'm disappointed the effort has now seemingly wasted. This has reminded me the importance of backups so I'll probably save a new copy after each day now.

I haven't tried another new city yet (was hoping this could somehow be saved, but... no? That circled area can't be all deleted and the issue goes away?), but I won't be using _Station Naming_ again because everything was renamed to the street it is in, police stations, schools, you name it. It made it hard to recognise what buildings were!

Guess it is a reason for looking forward to the upcoming update with something about improvements to water but for new cities not existing ones. But still, sigh... 

The question still remaining is: Why was I notified about the depreciated feature of lane connecting a year after it happened? This notification came up right before I couldn't open my city for the first time, the start of all this drama. Everything's fine, shut down computer, hours later start computer, reopen CS2, notification, open city, crash and never open the city again. Seems a bit of a strange coincidence, no?

**Reply by krzychu124** [reply]
> The question still remaining is: Why was I notified about the depreciated feature of lane connecting a year after it happened?

Are you 100% it was **before** you noticed that you can't load the saved game and not **after**? 
IMO, the game crashed in the previous session (e.g.: you tried loading corrupted savegame for the first time and game crashed) it likely also corrupted some mod settings files where TLE mod stores information that message was shown (it's not unusual that the game crash can corrupt mods settings files, though it's quite annoying). 

Like I mentioned, I've seen that message almost every time I restarted the game after crash caused by loading the corrupted saved game, including complete reset of game settings like audio volume, custom key bindings, graphic settings and all that. 

In `C:\Users\<your_username>\AppData\LocalLow\Colossal Order\Cities Skylines II` you should find file `C2VM-TrafficLightsEnhancement.coc`, which can be opened in notepad or any other text editor, then you should see inside:  `"m_HasReadLdtRetirementNotice": true`, which means you clicked the notification that redirects you **here**. 
If you remove this file, you will see the retirement notification again and it will go away only if you click on the notification, so the `.coc` settings file will be generated by the mod, with mentioned option set to `true`. 

Yes, it's a coincidence or rather indication that somehow the mod's settings file was reset or corrupted in the previous session.  

 > I only have one version of this city with 10 backups and they all crash. Hadn't worked on it too long, but long enough that I'm disappointed the effort has now seemingly wasted. This has reminded me the importance of backups so I'll probably save a new copy after each day now.

I'm not sure if I understand. You have 10 backups but they all not working? Does it mean they are all like.. autosaves from a single session - you didn't close the game or loaded any other saved game between the first and 10th backup?

**Comment by claudiomaggi19821982-creator** [comment]
grazie per l'aiuto fornito ma la mod per il traffico è rotta non compatibile con il mio cityskiline versione di gioco,ora comunque faccio sempre dei salvataggi ma continuo ad avere ogni tanto delle interruzioni di gioco causando ore di creazione perdute,magari è qualche altra mod ma skyve non rivela nulla al momento

**Comment by claudiomaggi19821982-creator** [comment]
grazie mille per la tua risposta,ho risolto eliminando asset e mod rotte,
 ora ho imparato a usare asset e mod sicure e il gioco funziona
benissimo,grazie ancora per la tua professionalità e risposta

Il giorno gio 16 ott 2025 alle 02:25 Krzysztof ***@***.***>
ha scritto:

**Comment by claudiomaggi19821982-creator** [comment]
ciao , ho fatto delle segnalazioni perché non conoscevo sia asset e mod che
causano i blocchi dei salvataggi e del gioco. la mod traffico funzionava
benissimo e anche io la uso per eliminare le invenzioni a U e le svolte
pericolose. come ti dicevo ho imparato a usare asset e mod corretti ora. se
ho fatto la segnalazione alla tua MOD mi scuso ma preso dal fatto che è un
gioco nuovo pensavo fosse la mod traffico a dare problemi,ma mi sbagliavo.
grazie amydella tua risposta 👋🏻✌️👍

Il giorno gio 16 ott 2025 alle 22:25 seoidau ***@***.***> ha
scritto:

**Comment by claudiomaggi19821982-creator** [comment]
come ti dicevo nei vecchi messaggi, il problema sono Asset e mod , bisogna
capire giocando quali mod o asset danno problemi, di solito lo si capisce
eliminando un po’ per volta ciò che causa il problema

Il giorno ven 17 ott 2025 alle 23:42 Krzysztof ***@***.***>
ha scritto:

**Reply by krzychu124** [reply]
It was not a replay to your comment. 
Please use Github instead of email if you want see or to add comments.
